workers/proof-manager/src/implementations/external_proof_manager/prover_service_client.rs

```diff
@@ -130,8 +130,7 @@ impl ProofServiceClient {
         let req = ValidWalletCreateRequest { statement: statement.clone(), witness };
         let res = self.send_request::<_, ProofResponse>(VALID_WALLET_CREATE_PATH, req).await?;
 
-        let link_hint = default_link_hint();
```

Comment:
ty for changing the `ProofBundle` type to only include these link hints for commitments & reblind, the usage of `default_link_hint` was a bit confusing before